src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+
+$block-width: 98px;
+$block-heigth: 98px;
+$entire-block: 100px;
+$gap: 1px;
+
+.block {
+  max-width: 726px;
+  min-width: 726px;
+  display: flex;
+  justify-content: left;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin-top: 30px;
+
+  &__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $block-heigth;
+    width: $block-width;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: transform 0.5s, background-color 0.5s;
+    cursor: pointer;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
```

Comment:
Fix everywhere
<img width="1063" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/49764881/3147d4ca-7185-4926-a65f-0265a50ecd19">

Reply:
I honestly don't understand what is the issue here. I have used nesting and used preuso-classes where they are applicable. My spacings also seems to be fine.